css/employees.css

```diff
@@ -0,0 +1,407 @@
+@import url("common.css");

```

Comment:
i have noticed some duplication css like `.sidebar-handle ` in employees.css, role-details, roles.css. 
Move the those common styles to common.css.
To encourage the reusability of styles across the pages.